src/CalcManager/CalcManager.vcxproj

```diff
@@ -134,9 +134,13 @@
   <PropertyGroup />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|Win32'">
     <GenerateManifest>false</GenerateManifest>
+    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\$(MSBuildProjectName)\</OutDir>
```

Comment:
This will interfere with official builds.

https://github.com/microsoft/calculator/blob/b6f9634e346e746517792367d8065d06d3900e82/build/pipelines/templates/build-single-architecture.yaml#L33

Probably what you want instead is to set GenerateProjectSpecificOutputFolder=true here in the project files, instead of in the MSBuild command line above.